bokeh/embed.py

```diff
@@ -40,19 +40,41 @@ def components(plot_object, resources=None):
               are **already loaded**.
 
     Args:
-        plot_object (PlotObject) : Bokeh object to render
-            typically a Plot or PlotContext
+        plot_objects (PlotObject|list|dict|tuple) : Will return script
+        and div element to render if PlotObject, or correlating structures
+        of script/div combinations
         resources : Deprecated argument
     Returns:
-        (script, div) : UTF-8 encoded
-
+        (script, div): UTF-8 encoded|list|dict|tuple
     '''
-    
+
     if resources is not None:
         warn('Because the ``resources`` argument is no longer needed, '
              'is it deprecated and will be removed in'
              'a future version.', DeprecationWarning, stacklevel=2)
     
+    if type(plot_objects) is list or type(plot_objects) is tuple:
```

Comment:
I would use `isinstance(x, (list, tuple))` instead of `type() is list or...`
and the same pattern for the other uses of type()...